### PR TITLE
#29 fix xsd e templates per validazione

### PR DESCRIPTION
**Pull request by taba90**
- fix cardinalità su xsd.

### CODE REVIEW

File: src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-armonized-service.xml [hunk 1]
```diff
@@ -212,7 +212,7 @@
       <srv:serviceType>
         <gco:LocalName>view</gco:LocalName>
       </srv:serviceType>
-      <srv:extent>
+      <gmd:extent>
```
Comment (etj) [inline src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-armonized-service.xml]: `srv:extent` is ok for services

File: src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-interoperable-service.xml [hunk 2]
```diff
@@ -211,7 +211,7 @@
       <srv:serviceType>
         <gco:LocalName>view</gco:LocalName>
       </srv:serviceType>
-      <srv:extent>
+      <gmd:extent>
```
Comment (etj) [inline src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-interoperable-service.xml]: `srv:extent` is ok for services

File: src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-invocable-service.xml [hunk 3]
```diff
@@ -212,7 +212,7 @@
       <srv:serviceType>
         <gco:LocalName codeSpace="http://inspire.ec.europa.eu/metadatacodelist/SpatialDataServiceType">other</gco:LocalName>
       </srv:serviceType>
-      <srv:extent>
+      <gmd:extent>
```
Comment (etj) [inline src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-invocable-service.xml]: `srv:extent` is ok for services

File: src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-network-service.xml [hunk 4]
```diff
@@ -212,7 +212,7 @@
       <srv:serviceType>
         <gco:LocalName>view</gco:LocalName>
       </srv:serviceType>
-      <srv:extent>
+      <gmd:extent>
```
Comment (etj) [inline src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-network-service.xml]: `srv:extent` is ok for services

File: src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-service.xml [hunk 5]
```diff
@@ -212,7 +212,7 @@
       <srv:serviceType>
         <gco:LocalName>view</gco:LocalName>
       </srv:serviceType>
-      <srv:extent>
+      <gmd:extent>
```
Comment (etj) [inline src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-service.xml]: `srv:extent` is ok for services

**Review by etj — CHANGES_REQUESTED** [review]

File: src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-armonized-service.xml (hunk 1 above)
Comment (taba90) [reply src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-armonized-service.xml]: fixed

File: src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-interoperable-service.xml (hunk 2 above)
Comment (taba90) [reply src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-interoperable-service.xml]: fixed

File: src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-invocable-service.xml (hunk 3 above)
Comment (taba90) [reply src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-invocable-service.xml]: fixed

File: src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-network-service.xml (hunk 4 above)
Comment (taba90) [reply src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-network-service.xml]: fixed

File: src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-service.xml (hunk 5 above)
Comment (taba90) [reply src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-service.xml]: fixed

**Comment by etj** [comment]
Superseded by #125

**Pull request closed without merging**